cmd/cmdtrace/cmd_span.go

```diff
@@ -130,7 +129,9 @@ func commandName(cmd *cobra.Command) []string {
 		}
 		name = append(name, c.Name())
 	}
-	sort.Sort(sort.Reverse(sort.StringSlice(name)))
+	for i, j := 0, len(name)-1; i < j; i, j = i+1, j-1 {
```

Comment:
can use `slices.Reverse` - which basically does the same, but will make the intent obvious